### ISSUE TITLE
how install this server ?

### ISSUE THREAD

**Issue by JuanJ67m** [svenhenrik/CardboardRemoteDesktop#4]
im new in ubuntu and i dont know how install this server , ¿is there some .deb version for install this server?

**Comment by mikeumus** [comment]
There is nothing mentioned for how to connect to a server in the app's description or here in a ReadMe or Wiki. It just says it's based on VNC Viewer?

@JuanJ67m, the answer is you go here: https://www.realvnc.com/products/vnc/
Install VNC free version, get the free license and then it'll give you the IP once the program is running. 
Hope it helps! :wave: :smile:

**Comment by Hadlock** [comment]
It's designed to connect to any VNC server, of which there are many for Linux, and other platforms including Windows. This is merely a compatible client. Getting a compatible VNC server downloaded and installed is outside the scope of this project.

I don't think there's any need for documentation on how to use the software, it works very similar to all other remote desktop software. To connect to a server simply click the yellow + to get to the "edit connection" menu, give it a name, put in the ip:port, and a password (if applicable).

**Comment by mikeumus** [comment]
Yes but it should at least be mentioned. Given at least one sentence. This is our first time trying to deal with VNCs.

**Comment by svenhenrik** [comment]
I'm afraid this project is a bit dormant as I don't have much time to work on it, but if you want to contribute I'd be happy to accept a good Readme.md pull request.

**Comment by zbyte64** [comment]
On ubuntu you can search your apps for "remote desktop" to configure VNC. For more detailed instructions go here: https://help.ubuntu.com/community/VNC/Servers

If the VNC client complains about encryption then run the following on the host: `gsettings set org.gnome.Vino require-encryption false`
